src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4667,6 +4667,38 @@ private void VisitInitializer(IOperation rewrittenTarget, ISymbolInitializerOper
             {
                 LeaveRegion();
             }
+	}
+
+        public override IOperation VisitEventAssignment(IEventAssignmentOperation operation, int? captureIdForResult)
+        {
+            var instance = operation.EventReference.Event.IsStatic ? null : operation.EventReference.Instance;
+            if (instance != null)
+            {
+                _evalStack.Push(Visit(instance));
+            }
+
+            IOperation visitedHandler = Visit(operation.HandlerValue);
+            IOperation visitedInstance = instance == null ? null : _evalStack.Pop();
+            var visitedEventReference = new EventReferenceExpression(operation.EventReference.Event, visitedInstance,
+                semanticModel: null, operation.EventReference.Syntax, operation.EventReference.Type, operation.EventReference.ConstantValue, IsImplicit(operation.EventReference));
+            return new EventAssignmentOperation(visitedEventReference, visitedHandler, operation.Adds, semanticModel: null,
+                operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
+        }
+
+        public override IOperation VisitRaiseEvent(IRaiseEventOperation operation, int? captureIdForResult)
```

Comment:
>VisitRaiseEvent [](start = 35, length = 15)

Can we assert that these are statements?

Reply:
Verified VB parses these as top level statements, added assert.